src/weather-bar.ts

```diff
@@ -212,16 +259,17 @@ export class WeatherBar extends LitElement {
       --color-windy: var(--color-sunny);
       --color-windy-variant: var(--color-sunny);
       --color-exceptional: #ff9d00;
+      --color-night: #333;
     }
     .bar {
-      height: 30px;
+      height: 48px;
```

Comment:
Please do not change this - since we won't support alternate icons, this should stay as-is.